rotkehlchen/chain/evm/tokens.py

```diff
@@ -97,6 +97,18 @@ def generate_multicall_chunks(
     return multicall_chunks
 
 
+def get_chunk_size_call_order(evm_inquirer: 'EvmNodeInquirer') -> tuple[int, list[WeightedNode]]:
```

Comment:
Add a docstring. Also move the `# skipping etherscan because chunk size is too big for etherscan` in that docstring